docs/source/docker/advanced/code_examples/python_run_active_learning.py

```diff
@@ -1,7 +1,38 @@
+import json
 import lightly
+from lightly.openapi_generated.swagger_client.models.dataset_type import DatasetType
 
 # Create the Lightly client to connect to the API.
-client = lightly.api.ApiWorkflowClient(token="TOKEN", dataset_id="DATASET_ID")
+client = lightly.api.ApiWorkflowClient(token="YOUR_TOKEN")
+
+# Create a new dataset on the Lightly Platform.
+client.create_dataset('pedestrian-videos-datapool', dataset_type=DatasetType.VIDEOS)
+
+# Pick one of the following three blocks depending on where your data is
+# AWS S3
+client.set_s3_config(
+    resource_path="s3://bucket/dataset/",
+    region="eu-central-1",
+    access_key="ACCESS-KEY",
+    secret_access_key="SECRET",
+    thumbnail_suffix=None,
```

Comment:
didnt we want to change this that by default thumbnails are added?

Reply:
maybe the code can be changed so that if this is not explicitly set to false, it defaults to `.lightly/thumbnails/[filename]_thumb.[extension]`?

Reply:
> didnt we want to change this that by default thumbnails are added?

Will do that in a separate check (we will have to go through all examples). Better is to just change default value instead and remove setting it to "None"

Reply:
Sure, do it in a followup issue. It just should be easy to pass the `purpose` argument without needing to fully specify the thumbnail_suffix but that the default would be taken